velox/substrait/tests/data/q6_first_stage.json

```diff
@@ -67,12 +67,21 @@
                         "input": {
                             "project": {
                                 "common": {
-                                    "direct": {}
+                                    "emit":{
+                                        "outputMapping":[
+                                           2
+                                        ]
+                                    }
```

Comment:
This change is another example to show how the existing plans using Velox/Substrait consumer may need to be updated to get the expected results. This is what is described in the description as well. @mbasmanova this was the failing test, I didn't catch this until I read the plan properly. The failing CI was because the emit was not properly applied. 

What it does is just output the project columns down the relation hierarchy.